install.org

```diff
@@ -4,46 +4,53 @@
+
+The bleeding-edge development version or Org is available at [[https://code.orgmode.org/bzg/org-mode][code.orgmode.org]].
+You can clone the repo via:
 #+BEGIN_SRC shell
 ~$ git clone https://code.orgmode.org/bzg/org-mode.git
 #+END_SRC
-Daily snapshots of the development version: [[https://orgmode.org/org-latest.tar.gz][tar.gz]] or [[https://orgmode.org/org-latest.zip][zip]].
 
-Worg will tell you more on the [[https://orgmode.org/worg/dev/org-build-system.html][build options]] and on how to [[https://orgmode.org/worg/org-faq.html#keeping-current-with-Org-mode-development][stay on the bleeding edge]].
+Daily snapshots of the development version are released as [[https://orgmode.org/org-latest.tar.gz][tar.gz]] or [[https://orgmode.org/org-latest.zip][zip]] files.
+
+Worg has documentation about [[https://orgmode.org/worg/dev/org-build-system.html][build options]] and about how to [[https://orgmode.org/worg/org-faq.html#keeping-current-with-Org-mode-development][stay on the bleeding edge]].
 
-One will also find a large number of packages to augment Org on [[https://melpa.org/#/?q=org-mode&sort=downloads&asc=false][MELPA]].
+There are also many packages that augment Org available on [[https://melpa.org/#/?q=org-mode&sort=downloads&asc=false][MELPA]].
 
 ** Get started
 
-When you're ready to start trying out Org, take a look at the [[file:quickstart.org][Quickstart]] page
-for an overview of the syntax.
+Once you're ready to Org, check out the [[file:quickstart.org][Quickstart]]
+page for an overview of the syntax.
 
 * Doom, Spacemacs, and popular configs
-The Emacs kits /Doom/ and /Spacemacs/ both come with default setups for Org --- i.e.
-Org and a selection of packages. Here's what you get with [[https://github.com/hlissner/doom-emacs/blob/develop/modules/lang/org/packages.el][Doom]], and likewise
-[[https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Bemacs/org/packages.el][Spacemacs]]. It's worth noting that /Doom/ installs a version of Org close to the
-development ~HEAD~.
+The Emacs starter kits /Doom/ and /Spacemacs/ come with a curated set of packages and configurations that may be
+more familiar to users coming from other editors (e.g. Vim). They include Org along with integrations between
+=org-mode= and other modes (such as =evil-mode=). Here are links to the current packages included with
+[[https://github.com/hlissner/doom-emacs/blob/develop/modules/lang/org/packages.el][Doom]], and
+[[https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Bemacs/org/packages.el][Spacemacs]].
+/Doom/ installs a version of Org that is close to the development ~HEAD~.
 
-Popular configs like [[https://github.com/purcell/emacs.d/blob/master/lisp/init-org.el][Purcell's emacs.d]] will also often include org-mode
-customisations, and Org-related packages.
+Popular configs like [[https://github.com/purcell/emacs.d/blob/master/lisp/init-org.el][Purcell's emacs.d]]
```

Comment:
Hmm. Does removing the "after" give the impression that Doom/Spacemacs just install packages and don't configure Org much? Maybe I'm being paranoid.

Reply:
Maybe prefixing it with something like "For an approach that isn't quite as invasive, popular configs ..."

Also reminder to include Scimax on the list of starter kits.

Reply:
Ah yes, Scimax also deserves a mention. One more niggle, I'm not a fan of the negative connotations of "invasive" (even though the denotation definately applies) - perhaps a term like "expansive" could be a improvement?

Reply:
Agree, not happy with invasive, will try to come up with a better way to make the distinction between starter kits and configs.

Reply:
I think the new version here is clearer. Can you double check it to make sure no negative language crept in?

Reply:
This reads far better to me, nice job :+1: